File: G1_PPAI/G1_PPA1_E1/Entidades/Encuesta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace G1_PPA1_E1.Entidades
{
    public class Encuesta
    {
        //Atributos
        private string descripcion;
        private List<Pregunta> pregunta;

        //Constructor
        public Encuesta(string descripcion, List<Pregunta> pregunta)
        {
            this.descripcion = descripcion;
            this.pregunta = pregunta;
        }
        //Metodos
        public string getDescripcionEncuesta()
        {
            return descripcion;
        }
        public bool esEncuestaDeCliente()
        {
            //Este metodo compara la descripcion de la coleccion de preguntas que se le hicieron al cliente con la descripcion de las preguntas de la encuesta CREO
            return true;
        }
        public void armarEncuesta()
        {
            //me mataste flaco
        }
    }
}

File: PPAI_E3/src/main/java/dsi/g1/entidades/Encuesta.java
package dsi.g1.entidades;

import java.util.List;

public class Encuesta {
    //atributos
    private String descripcion;
    private List<Pregunta> pregunta;

    //constructor

    public Encuesta(String descripcion, List<Pregunta> pregunta) {
        this.descripcion = descripcion;
        this.pregunta = pregunta;
    }

    //metodos

    public String getDescripcion() {
        return descripcion;
    }


}
